src/otx/core/model/entity/base.py

```diff
@@ -36,6 +37,8 @@ class OTXModel(nn.Module, Generic[T_OTXBatchDataEntity, T_OTXBatchPredEntity, T_
         num_classes: Number of classes this model can predict.
     """
 
+    _EXPORTED_MODEL_BASE_NAME = "exported_model"
```

Comment:
I would rather make it as parameter in export method to have possibility to modify this parameter when calling export method.

Reply:
Added to model.export()